.github/workflows/build.yml

```diff
@@ -43,10 +43,49 @@ jobs:
       if: "env.PUSH_VERSION_COMMIT != ''"
       run: git push origin ${{ github.event.ref }}
 
+  build_conda:
```

Comment:
Shy the additional step? Conda can simply be set up in the `build_unix` step itself, simplifying this a lot

Reply:
See my comment above.